monai/inferers/merger.py

```diff
@@ -152,12 +159,21 @@ def finalize(self) -> torch.Tensor:
 
         return self.values
 
+    def get_output(self) -> torch.Tensor:
+        """
+        Get the final merged output.
+
+        Returns:
+            torch.Tensor: merged output.
+        """
+        return self.finalize()
+
     def get_values(self) -> torch.Tensor:
         """
         Get the accumulated values during aggregation or final averaged values after it is finalized.
```

Comment:
Are the final values always averaged? Are there not other aggregation options?

Reply:
Yes, it is an average merger similar to `AvgMerger` but we can implement any other aggregation operations too, like min, max, etc. Are there anything in particular that you think would be commonly used? If we can have a list with priority we can plan to implement them after this.

Reply:
You might want to only take the maximum class prediction value (from C channel)

Reply:
If there are multiple values in the C channel and want to take the maximum, it can be done in the [patch] post-processing part by providing a function/transform. Also for the overlapping region we would need a `MaxMerger` which has yet to be implemented. https://github.com/Project-MONAI/MONAI/issues/5873

Reply:
I wonder whether an option to weight the aggregation (say, linearly) according to how far each pixel is from its patch border might help reduce border artefacts.